adsphd.cls

```diff
@@ -2271,6 +2437,72 @@ without written permission from the publisher.
 \let\orig@makebackcovergeneralX\makebackcovergeneralX
 \let\orig@makebackcovergeneralXII\makebackcovergeneralXII
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Provide back cover using the 2024 (XXIV) layout
+% used by the Arenberg Doctoral School for joint PhDs
+\newcommand{\makebackcoverXXIVjoint}{\makebackcovergeneralXXIVjoint{0mm}{1}}
+\ifadsphd@epub
+  \renewcommand{\makebackcoverXXIVjoint}{}
+\fi
+% \makebackcovergeneralXII{xskip}{clearpage}
+\newcommand{\makebackcovergeneralXXIVjoint}[2]{%
+  \typeout{adsphd.cls: Using default back cover page}
+  %
+  % Use Helvetica
+  \ifadsphd@helveticaneue
+  \else
+  \renewcommand{\sfdefault}{\helvetsfdefault}
+  \fi
+  %
+  % Only add empty pages if asked for
+  \ifthenelse{\equal{#2}{1}}
+  {
+    \cleardoublepage \mbox{}
+    \thispagestyle{empty}
+    \newpage
+  }{}
+  \thispagestyle{empty}
+  % 
+  % Blue bar on top
+  \begin{textblock*}{\adsphdpaperwidth+\lbleed+\rbleed}(#1-\rbleed,0mm)
+  \textblockcolour{kulprimaryblue}
+  \rule{0mm}{17mm}
+  \end{textblock*}
+  %
+  % Blue bar below logos
+  \begin{textblock*}{\adsphdpaperwidth+\lbleed+\rbleed}(#1-\rbleed,40mm)
+  \textblockcolour{kulprimaryblue}
+  \rule{0mm}{17mm}
+  \end{textblock*}
+  %
+  % Text box with address KU Leuven
+  \begin{textblock*}{\adsphdpaperwidth+\lbleed+\rbleed}(#1-10mm, 22mm)
+  \textblockcolour{}
+  \vspace{-5mm}
```

Comment:
For some reason, I need this correction to align the text with the text block of the parner university, but not sure why.